<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>802</width>
    <height>568</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pStartServerBtn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>99</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>start</string>
    </property>
   </widget>
   <widget class="QLabel" name="pServerStateLaber">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>server is running</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pStopServerBtn">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>99</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>stop</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="pCollectDataCheckBox">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>10</y>
      <width>131</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Collect datas</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="pConnectMsgBrowser">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>231</width>
      <height>391</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pClearConnectMsgBtn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>460</y>
      <width>99</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>clear</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>802</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
